packages/webdriver/src/utils.js

```diff
@@ -185,7 +185,7 @@ export function isW3C (capabilities) {
      *   (https://w3c.github.io/webdriver/#dfn-new-sessions)
      * - it is an Appium session (since Appium is full W3C compliant)
      */
-    const isAppium = capabilities.automationName || capabilities.deviceName
+    const isAppium = capabilities.automationName || capabilities.deviceName || (capabilities['device.name'])
```

Comment:
`appium.name` is experitest specific, feel free to add `appiumVersion`

Reply:
I did not get it. Do you mean device.name ? Shall I remove device.name and just test for appiumVersion. If this is the case ... Sure, will update the check.